src/views/Login.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="login-form">
+    <v-content>
+      <v-container fluid>
+        <v-row align="center" justify="center">
+          <v-col cols="12" sm="8" md="4">
+            <v-card elevation="12">
+              <v-toolbar class="login-form-toolbar" color="primary" flat>
+                <v-toolbar-title>Login</v-toolbar-title>
+              </v-toolbar>
+              <v-card-text>
+                <h4>Please select a realm to log in with</h4>
```

Comment:
Please pull feature `translations` into this branch and make this localizable

Reply:
ignore this

Reply:
You can actually do this work now as develop has the translations work

Reply:
@Ataww can we get this text translated please? Pop me a message if you need help with this